torch/utils/benchmark/utils/common.py

```diff
@@ -48,7 +48,7 @@ def title(self) -> str:
 
     def setup_str(self) -> str:
         return (
-            "" if self.setup == "pass"
+            "" if (self.setup == "pass" or not self.setup)
```

Comment:
I don't think I can, since it's a user defined string. It's kind of like an `Optional[str]`, except that `timeit.Timer` doesn't allow an empty `setup`. I agree it's slightly curious.

Reply:
oh, pass isn't a keyword, it's literally the Python code "pass". That's curious, but I guess matching timeit.Timer takes precedence here.